tools/packaging/hosting_debian_config.json

```diff
@@ -0,0 +1,35 @@
+{
+    "maintainer_name":"Microsoft",
+    "maintainer_email": "nugetaspnet@microsoft.com",
+
+    "package_name": "dotnet-hosting",
+    "install_root": "/usr/share/dotnet",
+
+    "short_description": "Microsoft ASP.NET Core Hosting Bundle 2.0.0",
+    "long_description": "ASP.NET Core Hosting Bundle enables hosting of ASP.NET Core applications and contains the ASP.NET Core Runtime Package Store and .NET Core Runtime. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/aspnet/home). We happily accept issues and PRs.",
+    "homepage": "https://www.asp.net/",
+
+    "release":{
+        "package_version":"0.0.0.0",
+        "package_revision":"1",
+        "urgency" : "low",
+        "changelog_message" : "Bootstrap loop package"
```

Comment:
Should try to understand what this field is... or leave it blank if we have nothing interesting to say?